pkg/sharing/secret_export_reconciler.go

```diff
@@ -47,11 +53,35 @@ func (r *SecretExportReconciler) mapSecretToExport(a handler.MapObject) []reconc
 	}
 }
 
+func (r *SecretExportReconciler) WarmUp() error {
```

Comment:
not totally intuitive what purpose warmup is serving so good candidate for a GoDoc --   gist of it is that otherwise we'd be event driven and wouldn't have pre-reconciled existing resources?    (there's a question somewhere in here along the lines of,  "but isn't kubernetes level-triggered?" )